package.json

```diff
@@ -1,35 +1,35 @@
 {
-
     "repository": {
         "name": "ironcore-getting-started",
         "version": "0.0.1",
         "private": false
     },
     "license": "MIT",
     "scripts": {
-      "lint": "eslint ./app",
-      "start": "webpack-dev-server --config webpack.config.js --colors --env.stage=tutorial",
-      "complete": "webpack-dev-server --config webpack.config.js --colors --env.stage=complete",
-      "testTutorial": "node ./nightwatch/scripts/local.runner.js -c nightwatch/conf/tutorial.conf.js -e chrome,firefox,edge,ie,safari",
-      "testComplete": "node ./nightwatch/scripts/local.runner.js -c nightwatch/conf/complete.conf.js -e chrome,firefox,edge,ie,safari"
+        "lint": "eslint ./app",
+        "start": "webpack-dev-server --config webpack.config.js --colors --env.stage=tutorial",
+        "complete": "webpack-dev-server --config webpack.config.js --colors --env.stage=complete",
+        "testTutorial": "node ./nightwatch/scripts/local.runner.js -c nightwatch/conf/tutorial.conf.js -e",
```

Comment:
We most likely want to add the specified browsers to run on back in here and on the line below.